static/views/admin/settings-developer-attributes.php

```diff
@@ -62,6 +62,11 @@
                 <span><?php _e( 'Attention! This attribute (slug) is used by Google Validator and Address Display functionality. It is set automaticaly and can not be edited on Property Adding/Updating page.', ud_get_wp_property()->domain ); ?></span>
               </div>
             <?php endif; ?>
+           <?php if($slug=="ID"): // for ID field we do not show Delete option ?>
```

Comment:
@raj-uimatic this is correct, but I probably would consider adding another attribute array like ```$wp_properties[ 'geo_type_attributes' ] ``` that contains all non-editable attributes. If I were really forward-thinking I would also add an `apply_filters()` to allow for that list of non-editable fields to be manipulated via a other logic in plugin.